source/codegen/proto_helpers.py

```diff
@@ -135,11 +135,13 @@ def get_message_parameter_definitions(parameters):
                 # type.
                 parameter_type = f'repeated {parameter["enum"]}'
             grpc_field_number = generate_parameter_field_number(parameter, used_indexes)
+            is_get_last_error_output_param = common_helpers.is_get_last_error_output_param(parameter)
```

Comment:
I noticed after posting the PR that our style guidelines don't allow for this line to be longer than 100 characters, so I need to shorten it.

Reply:
Oh yeah. For anything formatting I'd recommend installing Black to automatically reformat it. You can look at https://github.com/ni/grpc-device/wiki/Development-Tips-and-Tricks#black-format-on-save to see how to install and run it from a shell or as a vs-code extension.

Reply:
I shortened the line by changing the name of the field we write to the parameter definition to `is_get_last_error_output`.